label_studio/ml_models/migrations/0011_thirdpartymodelversion_model_provider_connection.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.25 on 2024-09-12 21:59
+
+from django.db import migrations, models, transaction
+import django.db.models.deletion
+import django_migration_linter as linter
+from core.redis import start_job_async_or_sync
+from ml_models.models import ThirdPartyModelVersion
+from ml_model_providers.models import ModelProviderConnection, ModelProviders
+
+
+def _fill_model_version_model_provider_connection():
+    for provider in [ModelProviders.OPENAI, ModelProviders.AZURE_OPENAI]:
+        this_provider_model_versions = ThirdPartyModelVersion.objects.filter(provider=provider).values('id', 'organization_id', 'provider_model_id')
+        for provider_model_version in this_provider_model_versions:
+            connection_ids = ModelProviderConnection.objects.filter(
+                organization_id=provider_model_version['organization_id'],
+                provider=provider,
+                **({'deployment_name': provider_model_version['provider_model_id']} if provider == ModelProviders.AZURE_OPENAI else {}),
+            ).values_list('id', flat=True)[:1]
+            connection_id = connection_ids[0] if connection_ids else None
+            print(f"id: {provider_model_version['id']}")
+            print(f"mpc id: {connection_id}")
```

Comment:
nit: I usually favor using `logger.info`

Reply:
accidentally left them in, removed